apps/framework-docs-v2/src/components/mdx/interactive/customize-panel.tsx

```
@@ -1,67 +1,234 @@
+
+  // Get global customizer state to avoid overlapping dialogs
+  // Use try-catch to handle SSG where provider isn't available
+  let guideSettings: {
+    showCustomizer: boolean;
+    isConfigured: boolean;
+    settings: any;
+    setShowCustomizer: (show: boolean) => void;
+  };
+
+  try {
+    guideSettings = useGuideSettings();
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)